lib/addStyles.js

```diff
@@ -43,10 +47,9 @@ var getElement = (function (fn) {
 		}
 		return memo[selector]
 	};
-})(function (target) {
-	return document.querySelector(target)
-});
+};
 
+var getElement;
```

Comment:
`getElement` => `element`

Reply:
Not sure about this. Because in this place we create function nor element and then use it right there https://github.com/webpack-contrib/style-loader/blob/master/lib/addStyles.js#L154